source/documentation/style-rules/declarations.liquid

```diff
@@ -142,15 +136,14 @@ $rounded-corners: false
   border-radius: if($rounded-corners, 5px, null)
 {% endcodeExample %}
 
-{% markdown %}
-## Custom Properties
-{% endmarkdown %}
+{{ '## Custom Properties' | markdown }}
 
 {% # Arguments are (in order): `dart`, `libsass`, `node`, `ruby`, optional feature name, additional details within %}
 {% compatibility true, '3.5.0', null, '3.5.0', 'SassScript Syntax' %}
-Older versions of LibSass and Ruby Sass parsed custom property declarations just like any other property declaration, allowing the full range of SassScript expressions as values. Even when using these versions, it's
-recommended that you use interpolation to inject SassScript values for
-forwards-compatibility.
+Older versions of LibSass and Ruby Sass parsed custom property declarations just
```

Comment:
The formatting changes here are a bit weird here, is this because of some auto-formatting?

Reply:
Yeah, in general I tried to stick to an 80 char limit throughout.